cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,28 @@
+class TimeLoggingPage {
+
+    assertPageTimeLogginDisplayed() {
+        return cy.url().should('include', '/time-logging');
+    }
+
+    getUserProfileButtonSelector() {
+        return cy.get('.user-info__title');
+    }
+
+    getNavListSelector() {
+        return cy.get('.main-nav');
+    }
+
+    getActiveNavMenuOptionSelector() {
```

Comment:
Here also method naming could be better, as you return an element not a selector

Reply:
fixed with commit 'f073c5'